packages/admin-ui/package.json

```diff
@@ -68,7 +68,7 @@
     "mkdirp": "^0.5.1",
     "nightwatch": "^0.9.21",
     "prettier": "^1.11.1",
-    "react-axe": "^3.0.1",
+    "react-axe": "^3.0.2",
```

Comment:
Is there a specific reason we need to depend on this specific patch release of `react-axe`? Otherwise, wouldn't it be enough to run `yarn upgrade`?

Reply:
Thank you for looking at this, and asking questions ...

but I think #528  is the way to keep things in sync